### DISCUSSION TITLE
Export/Import for Inventory Tags and Ground Items

### DISCUSSION THREAD

**Discussion by TheLope** [runelite/runelite#17959] [Ideas]
1. The ability to Export/Import Inventory Tags tags
2. The ability to Export/Import Ground Items highlights

These lists seem to already be accessible in the runelite profile, and can be manually shared/updated
![image](https://github.com/runelite/runelite/assets/150454754/000569e6-75ac-41e9-802f-b740bbc4ae06)

I would find it very helpful to be able to share these settings in a formal way, similar to how ground Ground Markers are handled.
I recognize these are not quite as simple as ground markers, nor are they as easy to understand what exactly you are sharing.

This would allow for common lists such as inventory tags for all melee weapons, clues which are wildy steps, etc.

**Comment by RonnyTommy548** [comment]
+support on this request, would be very useful for sharing tags for the current clue juggling meta
